core/core.mk

```diff
@@ -16,10 +16,6 @@ PLATFORM_FLAVOR_$(PLATFORM_FLAVOR) := y
 $(eval $(call cfg-depends-all,CFG_PAGED_USER_TA,CFG_WITH_PAGER CFG_WITH_USER_TA))
 include core/crypto.mk
 
-# Setup compiler for this sub module
```

Comment:
Now that compiler is setup from arm.mk, please add a comment above `include core/arch/$(ARCH)/$(ARCH).mk` stating that this include is supposed to setup the compiler.

Reply:
Sure